src/keys.rs

```diff
@@ -191,7 +191,16 @@ impl SpendValidatingKey {
         // so the point repr is the same as I2LEOSP of its x-coordinate.
         <[u8; 32]>::from(&self.0)
     }
+    /// Converts this spend validating key to its serialized form,
+    /// I2LEOSP_256(ak).
+    #[cfg(feature = "FROST")]
+    pub fn into_bytes(&self) -> [u8; 32] {
+        self.to_bytes()
+    }
 
+    /// Attempts to convert these bytes into a spend validating key
+    /// from its serialized form, I2LEOSP_256(ak). Returns None if
+    /// it can't be created.
     pub(crate) fn from_bytes(bytes: &[u8]) -> Option<Self> {
```

Comment:
In our crates we use the `visibility` crate to make things `pub` depending on a feature, so you could add

```suggestion
    #[cfg_attr(feature = "FROST", visibility::make(pub))]
    pub(crate) fn from_bytes(bytes: &[u8]) -> Option<Self> {
```

Reply:
@daira @nuttycom  @str4d, would you find this acceptable? Given that's a dependency, I'd be leveraging in this case I'd like to make sure you agree with going in such direction.

Reply:
@conradoplg would this work for entire `struct`s as well?

Because `NullifierDerivingKey` and some of its functions will have to be exposed as `pub` instead of `pub(crate)`. It's possible that the use of this visibility crate, although not documented enough, will be a good addition for what we need to achieve given that otherwise we would have to manually achieve the same and gate it through the `frost` feature config.

Reply:
`struct SpendValidatingKey` is already pub. Its wrapped field is not and should not be, but as it happens you can obtain it anyway as `svk.randomize(pallas::Scalar::zero())`.

Reply:
I added the visibility crate to implement the required changes on this PR